packages/backend-common/src/logging/rootLogger.ts

```diff
@@ -38,3 +26,35 @@ export function getRootLogger(): winston.Logger {
 export function setRootLogger(newLogger: winston.Logger) {
   rootLogger = newLogger;
 }
+
+export function createRootLogger(
+  options: winston.LoggerOptions = {},
+): winston.Logger {
+  const logger = winston.createLogger({
+    level: process.env.LOG_LEVEL || 'info',
+    format: winston.format.combine(
+      winston.format.errors({
+        stack: process.env.LOG_STACKTRACE !== undefined || false,
+      }),
+      process.env.NODE_ENV === 'production'
+        ? winston.format.json()
+        : coloredFormat,
+    ),
+    defaultMeta: {
+      service: 'backstage',
+    },
+    transports: [
+      new winston.transports.Console({
+        silent:
+          process.env.JEST_WORKER_ID !== undefined && !process.env.LOG_LEVEL,
+      }),
+    ],
+    ...options,
```

Comment:
loadsh.merge instead? so we don't have to repeat e.g. `defaultMeta.service`

Reply:
Yeah, I was thinking that a deep merge might not be desirable for some parts of the logger config. But, usually `transports` will be additive and the `format` is a class instance so can't be merged anyways. So yeah, I'll make this a lodash deep merge :)